drivers/platform/aducm3029/i2c.c

```diff
@@ -167,6 +172,8 @@ int32_t i2c_remove(struct i2c_desc *desc)
 		i2c_handler = NULL;
 		last_address = UNINITIALIZED_ADDRESS;
 		last_bitrate = UNINITIALIZED_BITRATE;
+		if (prologue_data)
+			free(prologue_data);
```

Comment:
Given it's a global I think it should be also set to NULL after freeing.

Reply:
Will do.